modules/spo-recording-profiles.adoc

```diff
@@ -108,22 +110,22 @@ I0517 13:55:36.383187  348295 enricher.go:376] log-enricher "msg"="audit" "conta
 +
 [source,terminal]
 ----
-$ oc -n openshift-security-profiles delete pod my-pod
+$ oc -n my-namepace delete pod my-pod
 ----
 
 . Confirm the Security Profiles Operator reconciles the two {type} profiles:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc get {object} -n my-namespace
+$ oc get seccompprofiles -lspo.x-k8s.io/recording-id=test-recording -n my-namespace
```

Comment:
This command only works for seccompprofile.
It will update https://62471--docspreview.netlify.app/openshift-enterprise/latest/security/security_profiles_operator/spo-selinux verification 2 also.

Reply:
Question for you @xiaojiey : Is the corrrect parameter of the "oc get" command to be called seccomprofiles or seccomprofile ? (keep the s at the end, or delete it?)

Reply:
Agreed and thank you! @xiaojiey . PTAL!